packages/react-components/source/react/library/button-select/ButtonSelect.js

```diff
@@ -35,6 +36,7 @@ const propTypes = {
   value: PropTypes.oneOfType([
     //eslint-disable-line
     PropTypes.string,
+    PropTypes.number,
```

Comment:
Should this also include an array of numbers?

Reply:
No i dont believe so, this value prop is the number or string that the user has selected so there should only be one value unless its a multi select which the button select dosn't support.